### ISSUE TITLE
LocationMessageSnapshotOptions are not accessible from delegate

### ISSUE THREAD

**Issue by Skyb0rg** [MessageKit/MessageKit#442]
## General Information

* MessageKit Version:
0.12.0
* iOS Version(s):
11.1
* Swift Version:
4
* Devices/Simulators:
both
* Reproducible in ChatExample? (Yes/No):
yes

##  What happened?

if i want to use LocationMessageSnapshotOptions in MessagesDisplayDelegate --->
```
func snapshotOptionsForLocation(message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> LocationMessageSnapshotOptions { 

return LocationMessageSnapshotOptions() 
```
**<--- Error: initializer is inaccessible due to 'internal' protection level**

`}`

## What did you expect to happen?

that i can use it ;)

what i did to make it working:
Edit
LocationMessageSnapshotOptions.swift

and add:

```
public init(showsBuildings: Bool = false, showsPointsOfInterest: Bool = false, span: MKCoordinateSpan = MKCoordinateSpan(latitudeDelta: 0, longitudeDelta: 0) , scale: CGFloat = UIScreen.main.scale) {
             self.showsBuildings = showsBuildings
             self.showsPointsOfInterest = showsPointsOfInterest
             self.span = span
             self.scale = scale
    }
```

Did i something wrong?
Thanks

**Comment by SD10** [comment]
@Skyb0rg Maybe we can just make the properties `public` and then the memberwise initializer will be public as well? I'd accept a PR for that

**Comment by zhongwuzw** [comment]
Fixed in #452
